source/vibe/web/rest.d

```diff
@@ -969,12 +987,13 @@ private string genClientBody(alias Func)() {
 				static if (is(PT == Json))
 					url_prefix = q{urlEncode(id.toString())~"/"};
 				else
-					url_prefix = q{urlEncode(toRestString(serializeToJson(id)))~"/"};
+					url_prefix = q{urlEncode(toRestString(id))~"/"};
```

Comment:
Why ?
I'm not a big fan of this line, but that'll break existing code. Unless it's required by this P.R. I suggest to create a separate one that tackles the Json problem (and gives motivation).